### DISCUSSION TITLE
macOS CLI missing/incomplete

### DISCUSSION THREAD

**Discussion by pejax** [ghostty-org/ghostty#3698] [Feature Requests, Ideas] [answered]
Ghostty help (ghostty --help) indicates that launching the terminal emulator from CLI is not supported under macOS. Without this feature a terminal emulator is useless for me - I need to open a terminal from scripts with the -e option to run commands. Other emulators I tried (WezTerm and Alacritty) have this feature. Is this planned for a future Ghostty release?

**Comment by 00-kat** [comment, marked as answer]
Currently you can use `open` to do so, e.g. `open -na "Ghostty" --args --someoption=somevalue`.

**Reply by rhodes-b** [reply]
as a note this is the planned way to do this, the docs should be better but I will marked this as solved

**Reply by 00-kat** [reply]
> the docs should be better

I PR'd a change to that a month or so ago, so the docs would be better once 1.2 releases.

**Comment by pejax** [comment]
Ah - very cool, thanks :-)
